src/sentry/testutils/fixtures.py

```diff
@@ -606,18 +609,19 @@ def create_full_event(self, event_id='a', **kwargs):
                                  data=json.loads(payload))
 
     def create_group(self, project=None, checksum=None, **kwargs):
+        assert 'message' not in kwargs, 'message was removed'
```

Comment:
```suggestion
        assert 'message' not in kwargs, 'message was removed use search_message instead'
```
Might help save folks some time when they make a mistake.